src/Style/Style.scss

```diff
@@ -80,6 +80,11 @@ div[id="content"]
 	height: 1.95em !important;
 }
 
+// Move the YouTube Slowmode countdown back over the Send button
+.yt-live-chat-message-input-renderer#countdown {
+	left: -73px;
```

Comment:
I'd prefer to avoid using pixels or any kind of absolute positioning. There must be a neater way to do this

Reply:
The solution is to place the 7tv button to the left instead of the right, the countdown's position is absolute so it will always show to the right.

Reply:
You could do what @Excellify said but looking at the CSS youtube uses, they use exact pixels there for it so I think that having exact pixels to shift it over is fine. AFAIK, and with some digging, with CSS, it is impossible to make it "neater". With JS it's most definitely possible but I don't think it's worth it at the end of the day.

Reply:
Oh youtube itself uses absolute pixel positioning. Guess that's fine then